app/design/frontend/Magento/storefront/Magento_Cms/templates/html/banner.phtml

```diff
@@ -10,3 +10,28 @@
         <img src="https://i.ibb.co/f1xtbTM/bannerimg.png" alt="">
     </div>
 </div>
+
+
+/*Categories*/
+<div class="categories">
+    <div class="categories_container">
+        <div class="cat_elem clock">
+            <img src="https://i.ibb.co/0JsvNrN/clock.png"/>
+            <div class="elem_textbox">
+                <h1>Clocks</h1>
+                <p>The 1958 moulded plywood armchair by Norman Cherner.</p>
+                <a href="#">view now</a>
+                <img src="https://i.ibb.co/0JsvNrN/clock.png"/>
+            </div>
+        </div>
+        <div class="cat_elem lamp">
+            <img src="https://i.ibb.co/M8SHWSq/lamp.png" alt="">
+            <div class="elem_textbox">
+                <h1><span>New</span> Lights</h1>
+                <p>The 1958 moulded plywood armchair by Norman Cherner.</p>
+                <a href="#">view now</a>
+                <img src="https://i.ibb.co/M8SHWSq/lamp.png" alt="">
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
Same comment as for 2/14 banner:
Please create a separate CMS block for categories and include it in the Homepage CMS page